src/haxelib/server/Repo.hx

```diff
@@ -80,6 +80,15 @@ class Repo implements SiteApi {
 		};
 	}
 
+	public function getLatestVersion( project : String ) : SemVer {
+		var p = Project.manager.select($name == project);
+		if( p == null )
+			throw "No such Project : "+project;
+
+		var vl = Version.manager.search($project == p.id, { orderBy: [-major, -minor, -patch, -previewNum], limit: 1});
```

Comment:
Same comment as for the previous version:
> I think this is not correct, since it doesn't account preview field that should be also considered for sorting. Not sure if it can be done within a simple SQL query.